src/input/input.jsx

```diff
@@ -12,7 +12,7 @@ export default function Input({ type, value, placeholder, onchange, width, heigh
         padding: padding,
         fontSize: fontSize,
         backgroundColor: backgroundColor,
-        color: color
+        color: color,
     }
 
     let icon = null;
```

Comment:
Do not use let

Reply:
because of in let we can assign then value inside of Let. That why I used this.